### PR TITLE
Add schemes to turn OCC commands

### PR DESCRIPTION
**Pull request by danxuliu**
Although it was possible to add TURN servers with OCC commands they had no scheme set, so in practice they always used "turn". If a different scheme had to be set that had to be done afterwards from the UI.

On the other hand, as no scheme could be set when deleting a TURN server all TURN servers matching the server and protocols were deleted regardless of their scheme.

Not sure if it is worth a backport or not, so feel free to call backportbot.

### CODE REVIEW

**Comment by danxuliu** [comment]
Oops, I did not realize that there were unit tests, I will fix them.

**Review by nickvergessen — APPROVED** [review]
This will break the commands in existing scripts, but I guess that is okay.

File: lib/Command/Turn/Add.php [hunk 1]
```diff
@@ -66,11 +70,16 @@ protected function configure(): void {
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$schemes = $input->getArgument('schemes');
 		$server = $input->getArgument('server');
 		$protocols = $input->getArgument('protocols');
 		$secret = $input->getOption('secret');
 		$generate = $input->getOption('generate-secret');
 
+		if (! in_array($schemes, ['turn', 'turns', 'turn,turns'])) {
```
[the remark this answers is not in the document]
Comment (danxuliu) [reply lib/Command/Turn/Add.php]: Done.

**Comment by nickvergessen** [comment]
Let's just fix it going forward.

**Comment by nickvergessen** [comment]
Btw while changing the arguments, we could also think about moving the schema and protocols to options?

**Pull request merged**